include/oneapi/dnnl/dnnl.hpp

```diff
@@ -1696,6 +1696,12 @@ struct memory : public handle<dnnl_memory_t> {
         ABcde8b16a = dnnl_ABcde8b16a,
         AcdeB8b16a = dnnl_AcdeB8b16a,
         AB8b8a = dnnl_AB8b8a,
+        abDC8d8c = dnnl_abDC8d8c,
```

Comment:
Out of interest, why do we need different memory formats for 8, 16 and 24? I think for similar kernels, ACL uses AB8b8a and trusts the prefetcher to pick up the 3 different streams. Is this something you considered? I'm just wondering if it would simplify the implementation.

Reply:
Thanks for explaining the reasoning, this is very useful. I don't personally have specific data either way, but it's good to know for future.